app/models/user.rb

```diff
@@ -0,0 +1,17 @@
+class User < ApplicationRecord
+  belongs_to :organization
+
+  validates :first_name, :last_name, presence: true, length: { in: 2..50 }
+  VALID_EMAIL_REGEX = /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i
```

Comment:
ordering! 
```
class Blah
  constants...

  modules...

  ...
end
```

Reply:
what about comment above? FIX it

read "rails code style" what should be first, second, ...